Gem/Code/Source/Weapons/WeaponTypes.cpp

```diff
@@ -363,7 +363,7 @@ namespace MultiplayerSample
 
     bool FireParams::operator!=(const FireParams& rhs) const
     {
-        return m_targetPosition.IsClose(rhs.m_targetPosition)
+        return !m_targetPosition.IsClose(rhs.m_targetPosition)
```

Comment:
Haha nice catch

Reply:
yeah, that actually caused target positions to not replicate correctly to simulated clients :(